### ISSUE TITLE
Rapid SDK v4.0 support?

### ISSUE THREAD

**Issue by brocktimus** [eWAYPayment/eway-rapid-ruby#2]
Just wondering if it would be possible to get support for v4.0 of the SDK? I notice it's already in the PHP version.

https://eway.io/api-v3/?shell#versions

It doesn't look like it would be hard since all the requests go via ```EwayRapid::Message::RestProcess```. Just opening an issue before I do so myself since it looks like at least some of the classes could have been templated originally.

**Comment by incarnate** [comment]
Hi @brocktimus 

Sorry for the lack of versioning love for the Ruby SDK! This is something I should be able to look at sometime next week, but there'd be no issue if you have the time to do it sooner as we aren't using templates any more.

Cheers!

**Comment by brocktimus** [comment]
FYI this ended up not being needed for the project I'm working on.

**Comment by incarnate** [comment]
Thanks for the update @brocktimus - I do have an update almost ready with this, just needs a final round of testing.

**Comment by incarnate** [comment]
Sorry for the massive delay on this, I've just published version 2.0.0 of the eWAY Ruby SDK with version support.  The version can be set using the following:

```
client = EwayRapid::RapidClient.new(API_KEY, PASSWORD, ENDPOINT)
client.set_version(40)
```

Merry Christmas!
